scss/bitstyles/ui/getting-started.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Getting started" />
+
+## Installing
+
+Install bitstyles using yarn or npm:
+
+```sh
+yarn add bitstyles
+
+# or
+
+npm install bitstyles
+```
+
+Bitstyles is a [Sass](https://sass-lang.com) library, so you’ll need to use Sass to build it. You could use the tools directly (see [Sass](https://sass-lang.com/install)). If you’re using webpack, use something like [sass-loader](https://github.com/webpack-contrib/sass-loader) for compiling.
+
+## Using bitstyles
+
+In your main Sass file (e.g. `index.scss`), import bitstyles using `@use`. The exact path you use will be dependent on your setup, but assuming a default setup and install location:
+
+```scss
+@use '~bitstyles/scss/bitstyles';
+```
+
+This will import the entire library, with default configuration.
+
+You may need to provide your sass installation with the path for bitstyles’ sass. This should be:
+
+```
+node_modules/bitstyles/scss
+```
+
+## Configuration
+
+Commonly you’ll want to change some configuration — colors, typography, sizes etc. When making changes to these global design tokens only, it can be done pretty simply in the `@use` statement:
+
+```scss
+@use '~bitstyles/scss/bitstyles' with (
+  $color-palette-text: #fff,
+  $color-palette-background: #000,
+  $layout-size-base: 1.2rem,
+  $margin-breakpoints: (
+    'l',
+    'xl',
+  )
+);
+```
+
+If you’re changing a lot of configuration, including component and utitlity classes, you may want to switch to configuring bitstyles on a module basis. A few things to note:
+
+- you’ll need to import every part of bitstyles individually, whether you’re overriding variables there or not
+- you can use that to selectively exclude modules you do not need, to keep your bundle size down
+- the variable names must lose the module-name prefix (e.g. `$color-palette-text` is now just `$text`, because the module’s scope is already clear)
+
+```scss
+@use '~bitstyles/scss/bitstyles/settings/color-palette' with (
+  $text: #fff,
+  $background: #000
+);
+@use '~bitstyles/scss/bitstyles/settings/layout' with (
+  $size-base: 1.2rem
+);
+
+// …
+
+@use '~bitstyles/scss/bitstyles/utilities/margin' with (
+  $breakpoints: (
+    'l',
+    'xl',
+  )
+);
+```
+
+## Legacy `@import`
+
+Sass still supports the `@import` directive. Though it’ll be deprecated and gone soon, bitstyles can for now still be configured and built using this method. It’s really recommended that you use the `@use` approach shown above though.
+
+```scss
+@import '~bitstyles/scss/bitstyles';
+```
+
+You can also configure bitstyles this way, by defining variables before importing. Note the variables need to be prefixed with `bitstyles-`:
+
+```scss
+$bitstyles-color-palette-text: #fff;
+$bitstyles-color-palette-background: #000;
+$bitstyles-layout-size-base: 1.2rem;
+$bitstyles-margin-breakpoints: ('l', 'xl');
+
+@import '~bitstyles/scss/bitstyles';
```

Comment:
does this way also work with `@use` ?

Reply:
This question also crossed my mind when I was writing this :D I didn’t try it yet, but I think it would work to do something like:

```scss
@use '~bitstyles/scss/bitstyles/settings/typography' with (
  $font-size-base: 10rem,
);

@use '~bitstyles/scss/bitstyles';
```

I’ll try it and see 💃

Reply:
Update: specifying the variables in the old way (with the `$bitstyles-` prefix) then `@use`ing the whole library doesn’t work.

My suggestion above also doesn’t work if you want to change configuration of components or utilities, only the global settings (the CSS is output at the point it’s `@use`d, so the source order is totally wrong)

Conclusion: I’ll add a section to explain the simpler way of importing when you only want to override global settings :+1: